pkg/feeds/npm/npm.go

```diff
@@ -179,11 +185,14 @@ func fetchAllPackages(registryURL string) ([]*feeds.Package, []error) {
 		uniquePackages[pkg.Title]++
 	}
 
+	n := 0
```

Comment:
Can this be replaced by a boolean flag `applyJitter := len(uniquePackage) > minJitterThreshold`

Reply:
Yes, good idea. Done